### PR TITLE
Changes data frame transforms to transforms in the attributes

### PR DESCRIPTION
**Pull request by szabosteve**
This PR changes `data frame transform` and its variations to `transform` and its respective variations in the attributes list.

### CODE REVIEW

File: shared/attributes.asciidoc [hunk 1]
```diff
@@ -155,9 +159,9 @@
 :cdataframes:                continuous {dataframes}
 :cdataframe-cap:             Continuous {dataframe}
 :cdataframes-cap:            Continuous {dataframes}
-:cdataframe-transform:       continuous {dataframe-transform}
-:cdataframe-transforms:      continuous {dataframe-transforms}
-:cdataframe-transforms-cap:  Continuous {dataframe-transforms}
+:cdataframe-transform:       continuous transform
```
Comment (szabosteve) [inline shared/attributes.asciidoc]:
```suggestion
:cdataframe-transform:       continuous {transform}
```

File: shared/attributes.asciidoc [hunk 2]
```diff
@@ -155,9 +159,9 @@
...
+:cdataframe-transforms:      continuous transforms
```
Comment (szabosteve) [inline shared/attributes.asciidoc]:
```suggestion
:cdataframe-transforms:      continuous {transforms}
```

File: shared/attributes.asciidoc [hunk 3]
```diff
@@ -155,9 +159,9 @@
...
+:cdataframe-transforms-cap:  Continuous transforms
```
Comment (szabosteve) [inline shared/attributes.asciidoc]:
```suggestion
:cdataframe-transforms-cap:  Continuous {transforms}
```

File: shared/attributes.asciidoc [hunk 4]
```diff
@@ -141,10 +141,14 @@
 :dataframes:                 data frames
 :dataframe-cap:              Data frame
 :dataframes-cap:             Data frames
-:dataframe-transform:        {dataframe} transform
-:dataframe-transform-cap:    {dataframe-cap} transform
-:dataframe-transforms:       {dataframe} transforms
-:dataframe-transforms-cap:   {dataframe-cap} transforms
+:dataframe-transform:        transform
```
Comment (lcawl) [inline shared/attributes.asciidoc]:
```suggestion
:dataframe-transform:        {transform}
```

File: shared/attributes.asciidoc [hunk 5]
```diff
@@ -141,10 +141,14 @@
...
+:dataframe-transform-cap:    Transform
```
Comment (lcawl) [inline shared/attributes.asciidoc]:
```suggestion
:dataframe-transform-cap:    {transform-cap}
```

File: shared/attributes.asciidoc [hunk 6]
```diff
@@ -141,10 +141,14 @@
...
+:dataframe-transforms:       transforms
```
Comment (lcawl) [inline shared/attributes.asciidoc]:
```suggestion
:dataframe-transforms:       {transforms}
```

File: shared/attributes.asciidoc [hunk 7]
```diff
@@ -141,10 +141,14 @@
...
+:dataframe-transforms-cap:   Transforms
```
Comment (lcawl) [inline shared/attributes.asciidoc]:
```suggestion
:dataframe-transforms-cap:   {transforms-cap}
```

File: shared/attributes.asciidoc [hunk 8]
```diff
@@ -155,9 +159,9 @@
 :cdataframes:                continuous {dataframes}
 :cdataframe-cap:             Continuous {dataframe}
 :cdataframes-cap:            Continuous {dataframes}
-:cdataframe-transform:       continuous {dataframe-transform}
-:cdataframe-transforms:      continuous {dataframe-transforms}
-:cdataframe-transforms-cap:  Continuous {dataframe-transforms}
+:cdataframe-transform:       continuous {transform}
```
Comment (lcawl) [inline shared/attributes.asciidoc]: Should we create "ctransform" attributes (or something like that) too?

Comment (szabosteve) [reply shared/attributes.asciidoc]: Yes, I'll add them.

**Comment by lcawl** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**